src/cloudant/design_document.py

```diff
@@ -15,6 +15,7 @@
 """
 API module/class for interacting with a design document in a database.
 """
+from cloudant.search import SearchIndex
```

Comment:
This is a nit but I like to keep the _2to3 module as the first module in our list of imported modules.  Since it is our Py2 to Py3 compatibility module.